### PR TITLE
Convert noteblocks for web/api folder (part 13)

### PR DESCRIPTION
**Pull request by queengooborg**
This PR converts the noteblocks for the 'web/api' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js). This is part 13.

### CODE REVIEW

**Comment by PassionPenguin** [comment]
Hi @queengooborg, this could have been done in #35143, #35144, #35145, #35146

**Comment by queengooborg** [comment]
Hey @PassionPenguin!  These new PRs that I am creating are from an existing working tree and an existing review process I have set up with @Josh-Cena.  Additionally, there are a number of cases where we decided to leave noteblocks unconverted for the time being due to Prettier formatting bugs and other reasons.

Thank you for submitting those PRs to help out with the migration, especially for translated content -- but if it's alright with you, I feel that we should continue with the PRs from my working tree, and then take a look at what's left from yours to tackle what was missed.  Does that sound good?

**Comment by queengooborg** [comment]
> b. It's absolutely Ok to continue with your work tree in case anything left behind for **/api**. Actually I had a doublecheck that all your previous editions were merged before I started editing mine. (Well sad for my editing for about some ten minutes. **I had closed those PRs**)

I had noticed the manual edits just a bit ago to fix the noteblocks that were already broken to begin with, like in #35151!  I think that's great, and we should definitely fix those -- would you be down to cherry-pick those fixes and submit a PR for them alone?  (Alternatively, a draft PR that combines all the other PRs together, which is then synced with main to produce only the differences between our changes, could also work!)

File: files/en-us/web/api/urlpattern/urlpattern/index.md [hunk 1]
```diff
@@ -37,7 +37,8 @@ new URLPattern(input, baseURL, options)
     - `search`
     - `hash`
     - `baseURL`
-      > **Note:** Omitted parts of the object will be treated as wildcards (`*`).
+      > [!NOTE]
+      > Omitted parts of the object will be treated as wildcards (`*`).
```
Comment (Josh-Cena) [inline files/en-us/web/api/urlpattern/urlpattern/index.md:41]:
```suggestion
    > [!NOTE]
    > Omitted parts of the object will be treated as wildcards (`*`).
```

**Review by Josh-Cena — APPROVED** [review]

**Comment by PassionPenguin** [comment]
> > b. It's absolutely Ok to continue with your work tree in case anything left behind for **/api**. Actually I had a doublecheck that all your previous editions were merged before I started editing mine. (Well sad for my editing for about some ten minutes. **I had closed those PRs**)
> 
> 
> 
> I had noticed the manual edits just a bit ago to fix the noteblocks that were already broken to begin with, like in #35151!  I think that's great, and we should definitely fix those -- would you be down to cherry-pick those fixes and submit a PR for them alone?  (Alternatively, a draft PR that combines all the other PRs together, which is then synced with main to produce only the differences between our changes, could also work!)

May I ask **if all changes of yours have been made?** (even though I had **checked** that with my 8PRs (2 of those merged, 6 closed) the repository should contain no any note-blocks except those exceptions, **I couldn't be sure if yours cover all the cases**)

In addition, I shall not be able to produce such a PR today due to a delayed flight (sad, have been stuck in the airport for some hours). I will try to have it opened maybe tomorrow.

**Comment by Josh-Cena** [comment]
Yes, we have verified via the regex `` > \*\*(Note|Warning):\*\*(?! [[{`_*]) ``. There are no cases left.

**Comment by PassionPenguin** [comment]
> Yes, we have verified via the regex `` > \*\*(Note|Warning):\*\*(?! [[{`_*]) ``. There are no cases left.

thats great :)

now it’s time for the localized moderators to worry about (found some thousands cases of old syntax in the translated-content)

**Pull request merged**